tensorflow/lite/tools/make/Makefile

```diff
@@ -206,7 +206,7 @@ $(OBJDIR)%.o: %.cc
 # For normal manually-created TensorFlow C++ source files.
```

Comment:
Just nit: "Tensorflow C++" -> "Tensorflow Lite C"? Also, change "Tensorflow C++" to "Tensorflow Lite C++" in line 202? Thanks!

Reply:
Done